### DISCUSSION TITLE
Quantum Computer Music Course [Stanford University] (2024-04-01)

### DISCUSSION THREAD

**Discussion by scottoshiro2** [Quantumland-art/community#55] [Events]
Hi everyone,

My name is Scott Oshiro. I just wanted to advertise my "Quantum Computer Music" that I will be teaching in person at Stanford University. It will run from April 1st through June 5th, 2024. Below is an overview of the course if you are interested in learning more about quantum computer music. If you are at Stanford this Spring quarter, please sign up for the in-person course. If there is a lot of interest here online (for those of you not attending Stanford), I can see if there is an option to audit the course online. If you have any questions please feel free to contact me at [soshiro@ccrma.stanford.edu](soshiro@ccrma.stanford.edu). 

**Quantum Computer Music (MUSIC 22A)**
[Course Listing URL](https://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&page=0&catalog=&academicYear=&q=Music+222A&collapse=)

In this course, students will get an introduction to quantum computing with a focus on developing computer music applications. We will then go into what quantum computing is, and why are researchers so excited about it. The building blocks and fundamentals of quantum computing (qubits, superposition entanglement, interference, etc.) will then be covered followed by sections on quantum algorithms and how they are used in current quantum computer music applications. In the later weeks, we will also cover quantum representations of digital audio signals along with a preview of different quantum hardware. There will be hands-on programming assignments along with weekly problem sets. Students will also be expected to build, complete and present a final quantum music generation application for this course.

**Comment by ntrocado** [comment]
Hi Scott! I'd be interested in auditing, if an online option is offered....
